python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -70,6 +70,17 @@
     db.Column("event_timestamp", db.types.TIMESTAMP),
 )
 
+
+MutablePartitionsTable = db.Table(
+    "mutable_partitions",
+    SqlEventLogStorageMetadata,
+    db.Column("id", db.Integer, primary_key=True, autoincrement=True),
+    db.Column("partitions_def_name", db.Text, nullable=False),
+    db.Column("partition_key", db.Text, nullable=False),
```

Comment:
if all our other tables use `partition` instead of `partition_key`, we should use `partition` here too. otherwise, either is fine